src/screens/benefit/BenefitFormScreenWrapper.tsx

```
@@ -0,0 +1,53 @@
+  useEffect(() => {
+    const fetchData = async () => {
+      if (!data.selectApiResponse && !data.schemaData && id) {
+        try {
+          const [selectResponse, userResponse] = await Promise.all([
+            getOne({ id, bpp_id }),
+            getUser()
+          ]);
+          setData({
+            selectApiResponse: selectResponse,
+            userData: userResponse,
+            context: state.context || { bpp_id, bpp_uri: import.meta.env.VITE_BAP_URL }
```

ESLint

[error] Replace `············context:·state.context·||·{·bpp_id,·bpp_uri:·import.meta.env.VITE_BAP_URL·}` with `↹↹↹↹↹↹context:·state.context·||·{⏎↹↹↹↹↹↹↹bpp_id,⏎↹↹↹↹↹↹↹bpp_uri:·import.meta.env.VITE_BAP_URL,⏎↹↹↹↹↹↹},`

(prettier/prettier)